src/graphql/helpers.ts

```diff
@@ -285,15 +284,6 @@ export function formatUser(user) {
   };
 }
 
-function isFlaggedProposal(proposal) {
-  if (flaggedLinks.length === 0) {
-    return false;
-  }
-
-  const flaggedLinksRegex = new RegExp(flaggedLinks.join('|'), 'i');
-  return flaggedLinksRegex.test(proposal.body) || proposal.flagged;
```

Comment:
This will now not show any notice on proposals with the spam links, unless we flag these proposal,

Reply:
Yes, we need to run a SQL query to retrospectively flag all past proposals containing flaggedLinks